src/index.ts

```diff
@@ -24,11 +24,26 @@ async function run() {
     })
   )
 
-  // 3) Post comment
   if (!inputs.skipEmpty || !planIsEmpty(plan)) {
-    await core.group('Render comment', () => {
-      const comment = renderComment({ plan, header: inputs.header })
-      return createOrUpdateComment({ octokit, content: comment })
+    const prContext = 'pull_request' in github.context.payload
```

Comment:
When writing GitHub action workflows, I typically use `github.event_name == 'pull_request'`. I think the equivalent check here would be:

```suggestion
    const prContext = github.context.eventName === 'push'
```

Reply:
pretty sure the equivalent would be pull_request no?

Reply:
See
https://docs.github.com/en/webhooks/webhook-events-and-payloads#push
> This event occurs when there is a push to a repository branch.
and
https://docs.github.com/en/webhooks/webhook-events-and-payloads#pull_request
> This event occurs when there is activity on a pull request

Reply:
Whoops, yes, of course, I meant "pull_request", sorry for the confusion :smile: